content/en/docs/setup/independent/install-kubeadm.md

```diff
@@ -174,6 +174,7 @@ systemctl enable --now kubelet
     EOF
     sysctl --system
     ```
+  - Make sure that the `br_netfilter` module is loaded before this step. To load it explicitly call `modprobe br_netfilter`.
```

Comment:
```suggestion
  - Make sure that the `br_netfilter` module is loaded before this step. This can be done by running `lsmod | grep br_netfilter`. To load it explicitly call `modprobe br_netfilter`.
```

The current text does not tell the user how to check if the module is currently loaded, added a suggestion for doing that.

Reply:
+1 @SalimGangji please click the "commit change" button above.